www/docs/providers/workos.md

```diff
@@ -0,0 +1,23 @@
+---
+id: workos
+title: WorkOS
+---
+
+## Documentation
+
+https://workos.com/docs/sso/guide
+
```

Comment:
Following #1955, can you add this section here, so we don't need to come back and update this file once your PR is merged? 😄 
```suggestion
## Options

The **42 School Provider** comes with a set of default options:

- [42 School Provider options](https://github.com/nextauthjs/next-auth/blob/main/src/providers/42.js)

You can override any of the options to suit your own use case.
```

Reply:
Done!